server/database/entities/users.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy.orm import Session
+from uuid import UUID, uuid4
+import dataclasses
+from server.database.schemas.users import UserCreate
+
+
+@dataclasses.dataclass
+class User:
+    id: UUID
+    email: str
+    hashed_password: str
```

Comment:
I see. But whether or not this exact class is a DTO is questionable. Entities must have all properties that are stored in a db, since they are basically pretty wrappers around models, which implement the basic object operation logic. Entities are not meant to be used outside the server. If DTOs convey information between different services, perhaps `database/schemas` are DTOs. Think of schemas as molds for the JSONs of our data objects which specify what properties of a model should be included in the JSON specifically for given operation. If you have any questions, let's discuss this in person, shall we?